src/account-info.js

```diff
@@ -23,6 +23,17 @@ function AccountInfo (object) {
   this._isMobileVerified = Boolean(object.sms_verified);
 
   this._currency = object.currency;
+
+  var notifications = {};
+  if (object.notifications_type) {
+    object.notifications_type.forEach(function (code) {
+      var type = Math.log2(code);
+      if (type === 0) notifications.email = true;
+      if (type === 2) notifications.http = true;
+      if (type === 5) notifications.sms = true;
```

Comment:
I think the first way is the easiest to understand, but if you disagree at least use indexOf in place of filter:
```
var notifs = object.notifications_type.map(Math.log2);
this._notifications = {
  email: Boolean(~notifs.indexOf(0)),
  http: Boolean(~notifs.indexOf(2)),
  sms: Boolean(~notifs.indexOf(5))
}
```